ejp_xml_pipeline/etl.py

```diff
@@ -39,11 +39,12 @@ def write_entities_in_parsed_doc_to_file(
 @contextmanager
 def get_opened_temp_file_for_entity_types(
         ejp_xml_data_config: eJPXmlDataConfig,
+        file_dir: str
 ):
-    with ExitStack() as stack, TemporaryDirectory() as tempdir:
+    with ExitStack() as stack:
         for ent_db_load_config \
                 in ejp_xml_data_config.entity_type_mapping.values():
-            ent_db_load_config.set_directory(tempdir)
+            ent_db_load_config.set_directory(file_dir)
```

Comment:
minor: Not part of this PR, but maybe it would be better to not update the config here. Maybe the directory could be passed to `get_full_file_location`.